app/gui/view/graph-editor/src/shortcuts.rs

```diff
@@ -62,7 +62,4 @@ pub const SHORTCUTS: &[(ensogl::application::shortcut::ActionType, &str, &str, &
     (Press, "debug_mode", "ctrl d", "debug_set_test_visualization_data_for_selected_node"),
     (Press, "debug_mode", "ctrl shift enter", "debug_push_breadcrumb"),
     (Press, "debug_mode", "ctrl shift up", "debug_pop_breadcrumb"),
-    (Press, "debug_mode", "ctrl n", "add_node_at_cursor"),
-    // === Execution Mode ===
-    (Press, "", "shift ctrl e", "toggle_execution_environment"),
```

Comment:
The sad thing that now merging this PR will make execution contexts impossible to use on Windows due to #6371 

However, I still like the simplification of the code.

Reply:
Perhaps we can add additional shortcuts to switch _to_ a specific environment? I imagine it should be a simple task. Like `ctrl+shift+l` for Live and `ctrl+shift+d` for design or whatever.

Reply:
Also, `shortcuts.md` needs to be updated

Reply:
Added two shortcuts as discussed.